eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml

```diff
@@ -124,6 +130,7 @@ stages:
       parameters:
         # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
         DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
+        PublishBinariesToRelease: true
```

Comment:
What does this do in the cases of binaries vs packages? Should we have a separate template for this scenario?

Reply:
It publishes them to the github release if there are any available, essentially.

I do think it needs to be either A) the same template or B) uses an output variable from the first template. Because I need to attach them to that specific git release.

Reply:
I'm mostly wondering because the template is mostly about publishing to the devops feed but I guess it also includes the github tagging/releasing as well.

Reply:
Nah I see where you're coming from. I'll update this to something a bit less contorted.